www/src/routes/student/ResumeReview.tsx

```diff
@@ -1,10 +1,26 @@
+import { useAuth0 } from '@auth0/auth0-react';
 import { Grid, Typography } from '@material-ui/core';
 import Button from '@material-ui/core/Button';
-import React, { FC } from 'react';
+import React, { FC, useEffect } from 'react';
 
 import UploadResumeIcon from '../../assets/add_resume.svg';
+import { useStudentDispatch, useStudentSelector } from '../../redux/substores/student/studentHooks';
+import getMyResumeReviews from '../../redux/substores/student/thunks/getMyResumeReviews';
 
 const ResumeReview: FC = () => {
+    const { resumeReviews } = useStudentSelector((state) => state.resumeReview);
+    const { getAccessTokenSilently } = useAuth0();
+    const dispatch = useStudentDispatch();
+
+    useEffect(() => {
+        dispatch(getMyResumeReviews({ tokenAcquirer: getAccessTokenSilently }));
+    }, []);
+
+    useEffect(() => {
+        // TODO: display upload button if there are no resume reviews
```

Comment:
Wouldn't this TODO be in the html you return?

Reply:
Yes this is cleaned up in #185